src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './App.css';
+
+import Calculator from './components/Calculator';
+
+function App() {
```

Comment:
As requested by the previous code reviewer, `All components should be class-based components` including your `app.js`

> * [ ]  Please use class-based component for the calculator as this is the major requirement of the project.
> 
> ### See the image for reference
> ![image](https://user-images.githubusercontent.com/83803414/164460170-00b4816e-cc40-4123-b871-c5db873b6d10.png)

Reply:
Hello @micheaol Thank you for the review. I have a concern here. App.ja is not part of the components. "Use class based components (All components should be class-based).
Once the Calculator is ready, import it and use it in your main component (App.js).
" Kindly read this statement again.

Reply:
Hi @white3d ,

Please note that `app.js` is part of your component, so it should be a class-based component, this is part of the project requirement.

I hope this helps.